### DISCUSSION TITLE
Electron Particle Weight in egs_chamber

### DISCUSSION THREAD

**Discussion by jstask0** [nrc-cnrc/EGSnrc#1090] [Questions] [answered]
Hi,
I'm writing a custom ausgab routine to record the energy deposited by each secondary electron separately. One thing I've noticed is that the particle weight of electrons in my test egs_chamber simulation is always less than 1 (it's around 1/100, but different for each electron). I've made sure that no variance reduction techniques are in use, as I thought something like cross-section enhancement would cause this. Why do the electrons have weight less than 1? I can provide the input file if needed.

**Comment by rtownson** [comment, marked as answer]
Generally cross section enhancement, particle splitting, russian roulette, and tmpPhsp would be the culprits. What source are you using, is it possible that it's a phase-space that was produced using brems splitting?

**Reply by jstask0** [reply]
Ah, I'm using an egs_collimated_source. I see now that that affects the particle weights. I have a follow up question then. I know that when calculating dose it is necessary to multiply the energy deposited by the particle's statistical weight. For my purposes, I need the energy deposited by the interactions of each secondary electron. Would it be correct to not multiply by the statistical weight in this case? I'm asking because it appears that the values I'm getting are off by a factor of 100, which would be resolved if I didn't multiply by the statistical weight of the particle. Or would it be better to choose a different source type that does not change the particle weights?

**Reply by rtownson** [reply]
What are you using for `distance`? It will affect the normalization. If you're modelling an isotropic source using the collimated source for speedup, then `d=1/(4pi)^(1/2)=0.28209`. It may be worthwhile to use an isotropic source just to compare. Also, note that the collimated source is an approximation - you will lose out on the contributions of particles that may have started out going "backwards" and then scattered forward toward your region of interest.
